packages/contract-schema/test/schema.js

```diff
@@ -3,20 +3,43 @@ var assert = require("assert");
 
 var MetaCoin = require("./MetaCoin.json");
 
-describe("Schema", function() {
-  it("validates correct input", function() {
+describe("Schema", function () {
+  it("validates correct input", function () {
     Schema.validate(MetaCoin);
   });
 
-  it("throws exception on invalid input", function() {
+  it("throws exception on invalid input", function () {
     var invalid = {
       abi: -1
     };
 
     try {
       Schema.validate(invalid);
     } catch (err) {
-      var abiErrors = err.errors.filter(function(error) {
+      var abiErrors = err.errors.filter(function (error) {
+        return error.dataPath === ".abi";
+      });
+      assert(abiErrors);
+    }
+  });
+
+  it("normalizes a correct input", function () {
+    Schema.normalize(MetaCoin, {
+      validate: true
+    });
+  });
+
+  it("throws exception when attempting to normalize an invalid schema", function () {
```

Comment:
Perhaps, "throws exception when attempting to validate invalid input during normalization"?